### DISCUSSION TITLE
Verifying one's own instance as an admin

### DISCUSSION THREAD

**Discussion by davetroy** [mastodon/mastodon#21425] [Q&A]
Potentially dumb question / user story:

As the admin of an instance, it would be nice to be able to list it in my profile as a website link, and have Mastodon verify me as the admin.

I thought "oh, maybe I can add a rel=me tag to the code" but that looks hard. And it also is unnecessary, since Mastodon already knows I'm a site admin.

So perhaps if a link to an instance where one is an admin is included in the bio items, it could be shown as verified (or even as verified as an admin)? Or am I thinking about this wrong?

Thanks.

**Comment by Deauthorized** [comment]
You mean a federated "instance admin" tag?

**Reply by davetroy** [reply]
Something like that. At minimum people visiting a user profile on an instance should probably have visibility (at the admin's discretion) into whether that person is an admin of that instance and wishes to verify their connection to it that way.

Same info is already shown on the `/explore` page, so it would seem reasonable to show it on the user's profile page too.
